.github/ISSUE_TEMPLATE/special_case_issue.md

```diff
@@ -0,0 +1,69 @@
+---
+name: Special Case Issue
+about: A complex or exceptional issue that doesn't fit the usual templates
+
+---
+
+<!-- 
+Please read the text in this edit field before filling it in. 
+If your issue doesn't neatly fit into either the bug report or feature request categories, and it involves a more complex or unique scenario, use this template. 
+Before proceeding, please review the existing templates to ensure that those do not better suit your needs:
+- Bug Report Template: https://github.com/nvaccess/nvda/issues/new?template=bug_report.md
+- Feature Request Template: https://github.com/nvaccess/nvda/issues/new?template=feature_request.md
+
+Issues may be closed if the required information is not present. 
```

Comment:
Let's give issue submitters a fair chance to provide required information, they aren't mind readers and may not know what is important for a developer. Probably should be something like "If required information is not provided when requested issues may be closed."